lib/modules/apostrophe-events-calendar-widgets/lib/api.js

```diff
@@ -0,0 +1,44 @@
+var moment = require('moment');
+var _ = require('lodash');
+
+module.exports = function(self, options) {
+  self.getEvents = async function(req) {
+    var month;
+    if (req.query && !req.body.month) {
+      month = moment().format('YYYY-MM');
+    } else {
+      month = req.body.month
+    }
+
+    var max = moment(month).add({ days: 15, months: 1 }).format('YYYY-MM-DD');
+    var min = moment(month).subtract(15, 'd').format('YYYY-MM-DD');
+    var query = { startDate: { $lte: max, $gte: min } }
+    var projection = _.merge((self.options.projection || {}), { startDate:1, title: 1, url: 1, type:1, slug:1 });
+    var events = await self.apos.modules[self.piecesModuleName].find(req, query, projection).toArray();
+
+    events.forEach(event => {
+      event.date = event.startDate      
+    });
+
+    var clndrOptions = {
+      template: self.render(req, '_clndr', {}),
```

Comment:
Why the underscore? Like a partial in symfony? Do we follow this convention anywhere else?

Reply:
I think that was the thinking .. if not obvious, happy to remove